ajax/libs/overpass/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "overpass",
   "version": "3.0.2",
-  "filename": "overpass-webfont/overpass.css",
+  "filename": "overpass-webfont/overpass.min.css",
```

Comment:
You can use `git rebase -i HEAD~2` and "edit" first commit.
When you finish first commit, you can use `git rebase --continue`.